test-automation/README.md

```diff
@@ -12,3 +12,18 @@ We use Postman to test our chat service where there are 4 different requests for
 
 ## User Service
 We used Postman for this service with 6 different requests. The test result can be found [here](https://github.com/bubblemans/2021-spring-cs160-team-EduAll/blob/userService2.0/test-automation/userService.postman_collection.json).
+
+## Course Service 
+We use Postman to test our chat service where there are 4 different requests for our basic REST APIs. You can find the test results [here](https://github.com/bubblemans/2021-spring-cs160-team-EduAll/blob/main/test-automation/courseService.postman_collection.json).
+
+## Frontend
+We use Selenium to test frontend. Before running the java file on the current test-automation folder, please download 
+
+selenium-server-standalone-3.9.1.jar
+selenium-server-3.9.1.zip
+chromedriver
+
+After that, you need to build path for the first two and create folder for the chrome driver
```

Comment:
Can you add more descriptions about how to run this test, especially how to run this using the command line? I will merge this first, and you can add more later. And you can include this to your updates for Friday's meeting.